Beta display card

bjlagman12 opened the pull request:
**Task Link:** https://synapsepay.monday.com/boards/180157847/pulses/371815143

**Description:**

DisplayCard

Added DisplayCard component into the dev-ui-docz

For the basic portion in the docz I added three DisplayCards that renders a regular card and active card with account information with a balance. As well as an active card that has no account information. Below the rendered component displays the code base for whats rendered above.

Props display all the necessary information the render the DisplayCard

**Updates:**
none

**Notes:**

none

**Visuals:**

![basic](https://user-images.githubusercontent.com/36527396/68627159-f0815c00-0491-11ea-85ab-b7f09a79edf0.png)
![props](https://user-images.githubusercontent.com/36527396/68627160-f0815c00-0491-11ea-8f19-1cb66e61f994.png)

colinritchey commented on src/pages/displaycard/DisplayCard.mdx:
```diff
@@ -0,0 +1,84 @@
+| buttonComponent         | `component`             | false     | The three dot button right of the title     |
+| innerButtonData         | `object`                | false     | Inner block button, data has to be empty    |
+| innerButtonData.text    | `string`                | false     | Button right of the title                   |
+| innerButtonData.onClick | `function`              | false     | Button right of the title                   |
```
Descriptions should be updated

bjlagman12 replied:
Updated the the description

bjlagman12 commented:
**Updates:**
- Updated the components for the "Basics"
- Added components rendering "Absent Data"
- Prop description for the innerButtonData


**Visuals:**
![DPC basic 2](https://user-images.githubusercontent.com/36527396/68713005-bd9e9d00-0551-11ea-85d2-ff025968e65a.png)
![DPC absent](https://user-images.githubusercontent.com/36527396/68713010-c000f700-0551-11ea-9848-84d8def7cc77.png)
